docs/tsc/meetings/2019-03-11.md

```diff
@@ -0,0 +1,85 @@
+        - Some of the major DCCs using OCIO are on Windows and MacOS which could prove problematic.
+        - Do we need to build all the dependencies first, and then build OCIO against them.
+    - It should be of the style
+        - Pull the dependencies, rather than using the archive files found in the "ext/" directory.
+        - Maybe also have a flag to say use my local build of the library.
+        - Run CMake and build.
+    - What is the motivation behind doing this?
+        - This maybe a good opportunity to address the technical debt.
+        - John: Happy to help connecting with the other project coordinators.
+        - Mark: Uncertain that it is actually technical debt and more of a purest point of view in that we don’t want code from other sources checked into the OCIO repository?
+    - It would be good to at least gain a better understanding of the dependencies.
+    - Some of the current dependencies have raised red flags.
+    - The LCMS color management repository has been abandoned. Is currently only used for baking ICC profiles.
+    - Mark: Happy to go away and fix up what he can.
+    - Larry: Raised that it may lead to security issues if we don’t do this. Old versions can cause problems. OpenEXR had this problem in the past.
+    - John: Mavin solved the dependency fetch for Java a while ago. We should look to solutions like this.
+    - Concerns we raised in that CMake is not the best at pulling in dependencies from the internet.
+    - Patrick: We are not just able to download binaries, but we will also need to download the source of specific versions and build locally on the machine building OCIO. We will also have to be careful about static libraries and symbols. We will have to look at name-spacing things carefully.
+    - We will need to make the TAC aware of any external code that is needed when we ship with the library.
+
+* CLA/DCO requirements
+    - Needs them signed before doing merges into the new repository.
+    - John: Resolving issues with the initial setup. Team is currently backlogged on other projects but hoping to get something done in the next couple of days.
+    - DCO when we are ready to DCO sign off with commits.
+    - Every commit will have DCO sign off.
+    - John: Some info can be pulled or referenced from the TAC guidelines.
+    - Larry: Who needs to sign off the work?
+        - John: We can refer to ​https://developercertificate.org/
+    - Michael: This isn’t required until we move to the new account. Not to worry about it for now.
+    - Want to have the Linux foundation CLA system up and running first, pre-sign all the CLAs, and do one commit to flick the switch.
+    - Larry: Have agreed on the CLA but not on all the surrounding areas e.g. signature blocks etc.
+
+* Googlegroup archiving & updated gh-pages deployment
+    - Announced we will pull down the group in a week.
+    - A script will be run to get old contend and re-import it into the new group.
```

Comment:
```suggestion
    - A script will be run to get old content and re-import it into the new lists.
```

Reply:
Correction made in 519694f4b44c3d9a5e57f5084f221329b4734236